admin/data/sql/migrations/20200210153633.do.update-sp-create-checks-pt-1.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE mtc_admin.checkPin ADD pinValidFromUtc [datetimeoffset](3) NOT NULL;
+EXEC sp_rename 'mtc_admin.checkPin.pinExpiresAt', pinExpiresAtUtc, 'COLUMN';
```

Comment:
Surely this is best left alone?

Reply:
@GeorgeChatzigiannis you have misunderstood the request - we want the object in the payload to have the Utc postfix, not the database column